doc/website/docs/quick-start.md

```diff
@@ -26,23 +25,18 @@ You can download Community Edition from [Releases](https://github.com/ant-media
 
 Open a terminal and go to the directory where you download the Ant Media Server.
 
-
 ```shell
 cd /path/to/where/ant-media-server/is/downloaded
 ```
 
 Get the `install_ant-media-server.sh` script.
 
-
-
 ```shell
-wget https://raw.githubusercontent.com/ant-media/Scripts/master/install_ant-media-server.sh 
-chmod 755 install_ant-media-server.sh
+wget https://raw.githubusercontent.com/ant-media/Scripts/master/install_ant-media-server.sh & chmod 755 install_ant-media-server.sh
```

Comment:
In shell script, it's logical && operator, not just single &.
```wget https://raw.githubusercontent.com/ant-media/Scripts/master/install_ant-media-server.sh && chmod 755 install_ant-media-server.sh```

Reply:
you are right, we should use double ampersand